install/linux/usr/bin/odemis-yaml-selector

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# change search dir to the target directory where the main files are placed
+search_dir='/usr/share/odemis/'
+# adjust the search pattern to match a unique filename identifier
+search_pat='*meteor-main*.yaml'
+mic_list=()
+# keep track of a counter to store the index of the user selection
+counter=0
+
+# find all the files matching the search pattern and store them in an array
+files=($(find $search_dir -name $search_pat -exec basename {} \;))
+
+for fn in ${files[@]};
+do
+    # extract the microscope name by using this small Python script
+    # note: does not check for errors when parsing
+    #       spaces in the microscope names cannot be parsed
```

Comment:
It should handle spaces in names. 
In bash, it's a bit of a pain, but usually putting `"` before the variable tells bash you want to keep it a single word.
Here, you can write 
`mic_name="$(python3 -c "import sys; import yaml; from odemis.odemisd import modelgen; f = open('$search_dir$fn'); d = yaml.load(f, modelgen.SafeLoader); print(next(n for n, d in d.items() if d.get('class') == 'Microscope'))")"`